src/diagram/diagram-api.tsx

```diff
@@ -8,11 +8,13 @@ export interface DiagramActions {
 
   getGroups(): Group[];
 
-  addGroup(group: Group): void;
+  addGroup(group: Group, content?: string[]): void;
 
   removeGroups(groups: string[]): void;
 
-  extendGroup(group: Group, extension: string[]): void;
+  setGroup(group: Group, content: string[]): void;
+
+  getGroupContent(group: Group): string[];
```

Comment:
Add new getGroupContent because currently there was no way to query the group content outside of editor.